Sources/CountLabel.swift

```diff
@@ -48,15 +48,15 @@ open class CountLabel: UILabel {
         self.timer = nil
 
         if duration <= 0.0 {
-            self.setTextValue(self.endValue)
-            self.completion?()
+            setTextValue(endValue)
+            completion?()
         }
 
-        self.easingRate = Double(abs(endValue - startValue)).map(0 ... 9999, 1.1 ... 3.0)
-        self.span = abs(endValue - startValue)
-        self.progress = 0
-        self.totalTime = duration
-        self.lastUpdate = Date.timeIntervalSinceReferenceDate
+        easingRate = Double(abs(endValue - startValue)).map(0 ... 9999, 1.1 ... 3.0)
+        span = abs(endValue - startValue)
+        progress = 0
+        totalTime = duration
+        lastUpdate = Date.timeIntervalSinceReferenceDate
 
         let timer = CADisplayLink(target: self, selector: #selector(updateValue))
```

Comment:
So I guess I need to use self here cause of this local property, but what is the best practise? To give the local property another name, or to use self for the stored property?